src/tests/TestsRunHost.ts

```diff
@@ -3,23 +3,26 @@
 // ----------------------------------------------------------------------------
 'use strict';
 
+import {
+    runTests
+} from '@vscode/test-electron';
 import * as path from 'path';
```

Comment:
💡 Just as a note: I see this kind of really solves the mystery for me, essentially all these tests already existed, and upon adding the test runner and coverage we get 63% coverage for free, which is a win! 🐴🚀🔔